### PR TITLE
Fix xkit control panel to stay on-screen on mobile devices

### PR DESCRIPTION
**Pull request by dlmarquis**
It's not super-optimized, just some css to make it usable, but we need to rewrite pretty much all the desktop HTML before I'm going to think about optimizing it further.

### CODE REVIEW

**Comment by nightpool** [comment]
I don't have an android device to test design with ATM—can you post screenshots?

**Comment by dlmarquis** [comment]
http://imgur.com/bjXSdlY
http://imgur.com/ONB1K5m
http://imgur.com/tpxD9Zp

It's just the old panel, but with div scrolling. The tabs and content divs scroll separately, and the right pane (ext info) is set to be 100% the width of the viewscreen such that you can flick back and forth from the left menu to the settings.

The other tabs look like they do in desktop. I want to redo the extension gallery in the future so that each extension doesn't take up so much space, but the html for those is awful and I was going to deal with that when I went to overhaul the html.

The weird text wrapping in the settings is also something I couldn't fix without overhauling the html, but I might be overlooking something. If you have any ideas on how to fix it, let me know.

File: Extensions/xkit_preferences.js [hunk 1]
```diff
@@ -39,6 +39,29 @@ XKit.extensions.xkit_preferences = new Object({
...
+				'#xkit-extensions-panel-left { height: 100% !important; top: 0; left: 0; } ' +
+				'.xkit-extensions-display-type-switcher { bottom: 0; } #xkit-extensions-panel-left-search { left: 0; bottom: 0; } ' +
+				'#xkit-extensions-panel-right { height: calc(100% - 40px); width: 100%; } ' +
+				'#xkit-extensions-panel-right.xkit-wide-panel { left: 0; width: 100%; } ' +
+				'#xkit-extensions-panel-top { min-height: 100px; height: unset; } ' +
+				'#xkit-extensions-panel-top .buttons, #xkit-extension-enabled, #xkit-extension-internal-label, #xkit-extensions-panel-top .more-info, #xkit-extensions-panel-top .version, #xkit-extensions-panel-top .title { display: block; position: relative; right: unset; bottom: unset; top: unset; } ' +
+				'#xkit-extension-enabled { top: 5px; }' +
+				'.xkit-change-ext-setting-checkbox { font-size: 15px !important; bottom: 7px; top: 5px; }  .xkit-change-ext-setting-checkbox b { position: relative; bottom: -3px; } ' +
+				'.xkit-extension-setting .checkbox { height: unset !important; min-height: 30px; } ' +
+				'.xkit-extension-setting .title { font-size: 15px !important; } .xkit-extension-setting { padding: 10px 15px 10px 15px !important; } ' +
```
Comment (hobinjk) [inline Extensions/xkit_preferences.js]: .title should have `position: relative` on it to make it behave as an inline label

Comment (hobinjk) [inline Extensions/xkit_preferences.js]: Probably `position: relative !important`

**Comment by hobinjk** [comment]
LGTM @homu r+

**Pull request merged**